backend/models/dtos/workspace_imagery_dto.py

```diff
@@ -0,0 +1,15 @@
+from schematics import Model
+from schematics.types import (
+    StringType,
+    IntType,
+    ListType,
+    UTCDateTimeType,
+    UUIDType,
+)
+
+class WorkspaceImageryDTO(Model):
+    workspace_id = IntType()
+    definition = ListType(StringType())
```

Comment:
The type seems off, no?

```suggestion
    definition = ListType(DictType())
```